core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -2995,126 +4253,69 @@ public void testWindowSpecification()
                                                 Optional.empty(),
-                                                ImmutableList.of(new Identifier("a")),
+                                                ImmutableList.of(new Identifier(location(1, 52), "a", false)),
                                                 Optional.empty(),
                                                 Optional.empty())),
-                                new WindowDefinition(
-                                        new Identifier("otherWindow"),
-                                        new WindowSpecification(
-                                                Optional.of(new Identifier("someWindow")),
+                                new WindowDefinition(location(1, 56),
+                                        new Identifier(location(1, 56), "otherWindow", false),
+                                        new WindowSpecification(location(1, 72),
+                                                Optional.of(new Identifier(location(1, 72), "someWindow", false)),
                                                 ImmutableList.of(),
-                                                Optional.of(new OrderBy(ImmutableList.of(new SortItem(new Identifier("b"), ASCENDING, UNDEFINED)))),
+                                                Optional.of(new OrderBy(location(1, 83), ImmutableList.of(
+                                                        new SortItem(location(1, 92), new Identifier(location(1, 92), "b", false), ASCENDING, UNDEFINED)))),
                                                 Optional.empty()))),
                         Optional.empty(),
                         Optional.empty(),
                         Optional.empty()));
     }
 
+    @Test
     public void testUpdate()
     {
-        assertStatement("" +
-                        "UPDATE foo_table\n" +
-                        "    SET bar = 23, baz = 3.1415E0, bletch = 'barf'\n" +
-                        "WHERE (nothing = 'fun')",
-                new Update(
-                        new NodeLocation(1, 1),
-                        table(QualifiedName.of("foo_table")),
-                        ImmutableList.of(
-                                new UpdateAssignment(new Identifier("bar"), new LongLiteral("23")),
-                                new UpdateAssignment(new Identifier("baz"), new DoubleLiteral("3.1415")),
-                                new UpdateAssignment(new Identifier("bletch"), new StringLiteral("barf"))),
-                        Optional.of(new ComparisonExpression(ComparisonExpression.Operator.EQUAL, new Identifier("nothing"), new StringLiteral("fun")))));
+        final String updateSql = "" +
+                "UPDATE foo_table\n" +
+                "    SET bar = 23, baz = 3.1415E0, bletch = 'barf'\n" +
+                "WHERE (nothing = 'fun')";
+
+        assertThat(statement(updateSql)).isEqualTo(new Update(
```

Comment:
Format as follows to make it easier to read:

```
        assertThat(statement(updateSql))
                .isEqualTo(new Update(
                    ....
```

The same goes for all other uses of `.isEqualTo`. See how we do it in TestTypeParser.